src/_includes/docs/debug/debug-flow-android.md

```diff
@@ -0,0 +1,83 @@
+{{site.alert.note}}
+  If you want to use the [GNU Project Debugger][] to debug the
+  Flutter engine running within an Android app process,
+  check out [`flutter_gdb`][].
+{{site.alert.end}}
+
+[GNU Project Debugger]: https://www.sourceware.org/gdb/
+[`flutter_gdb`]: https://github.com/flutter/engine/blob/main/sky/tools/flutter_gdb
+
+#### Build the Android version of the Flutter app in the Terminal
+
+To generate the needed Android platform dependencies,
+run the `flutter build` command.
+
+```terminal
+$ flutter build appbundle --debug
+```
+
+```terminal
+Running Gradle task 'bundleDebug'...                               27.1s
+✓ Built build/app/outputs/bundle/debug/app-debug.aab.
+```
+
+{% comment %} Nav tabs {% endcomment -%}
+<ul class="nav nav-tabs" id="vscode-to-android-studio-setup" role="tablist">
+    <li class="nav-item">
+        <a class="nav-link active" id="from-vscode-to-android-studio-tab" href="#from-vscode-to-android-studio" role="tab" aria-controls="from-vscode-to-android-studio" aria-selected="true">Start from VS Code</a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link" id="from-android-studio-to-vscode-tab" href="#from-android-studio-to-vscode" role="tab" aria-controls="from-android-studio-to-vscode" aria-selected="false">Start from Android Studio</a>
+    </li>
+</ul>
+
+{% comment %} Tab panes {% endcomment -%}
+<div class="tab-content">
+
+<div class="tab-pane active" id="from-vscode-to-android-studio" role="tabpanel" aria-labelledby="from-vscode-to-android-studio-tab" markdown="1">
+
+#### Start debugging with VS Code first {#from-vscode-to-android-studio}
```

Comment:
Do you plan to hide this (and similar headers) from the right TOC to work around the issues with it showing even the hidden headers?

```suggestion
#### Start debugging with VS Code first {#from-vscode-to-android-studio}
{:.no_toc}
```